pkgs/applications/science/math/R/default.nix

```diff
@@ -75,6 +83,7 @@ stdenv.mkDerivation (finalAttrs: {
       FC="${gfortran}/bin/gfortran" F77="${gfortran}/bin/gfortran"
       JAVA_HOME="${jdk}"
       RANLIB=$(type -p ranlib)
+      CURL_CONFIG="${lib.getExe' (lib.getDev curl) "curl-config"}"
       r_cv_have_curl728=yes
```

Comment:
Is setting `r_cv_have_curl728` still needed?

Reply:
Not anymore, it can be removed.

Reply:
Though I did not check whether it would be required for a cross build. (texlive does not cross compile)